### PR TITLE
zh-translation: /best-practices/traffic-management/index.md #1731

### PR DESCRIPTION
**Pull request by gorda**
- [x] Docs

Ref: [ServiceMesher#1731](https://github.com/servicemesher/istio-official-translation/issues/1731)

### CODE REVIEW

File: content/zh/docs/ops/best-practices/traffic-management/index.md [hunk 1]
```diff
@@ -265,57 +216,40 @@ metadata:
...
-1. Although the order of evaluation for rules in any given source `VirtualService` will be retained,
-   the cross-resource order is UNDEFINED. In other words, there is no guaranteed order of evaluation
-   for rules across the fragment configurations, so it will only have predictable behavior if there
-   are no conflicting rules or order dependency between rules across fragments.
-1. There should only be one "catch-all" rule (i.e., a rule that matches any request path or header) in the fragments.
-   All such "catch-all" rules will be moved to the end of the list in the merged configuration, but
-   since they catch all requests, whichever is applied first will essentially override and disable any others.
-1. A `VirtualService` can only be fragmented this way if it is bound to a gateway.
-   Host merging is not supported in sidecars.
+1. 尽管会保留任何给定源 `VirtualService` 中规则的评估顺序，但跨资源的顺序是 UNDEFINED。换句话说，无法保证片段配置中规则的评估顺序，因此，只有在没有规则冲突或片段之间规则之间没有顺序依赖性时，它才具有可预测的行为。
```
Comment (yuxiaobo96) [inline content/zh/docs/ops/best-practices/traffic-management/index.md]:
```suggestion
1. 尽管会保留任何给定源 `VirtualService` 中规则的评估顺序，但跨资源的顺序是 UNDEFINED。换句话说，无法保证片段配置中规则的评估顺序，因此，只有在片段规则之间没有冲突的规则或者顺序依赖性时，它才具有可预测的行为。
```

Comment (gorda) [reply content/zh/docs/ops/best-practices/traffic-management/index.md]: @yuxiaobo96 感谢细心 review，已修改

**Pull request merged**